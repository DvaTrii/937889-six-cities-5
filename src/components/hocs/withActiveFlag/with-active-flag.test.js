import React from "react";
import renderer from "react-test-renderer";
import {withActiveFlag} from "./withActiveFlag";

const noop = () => {};

const MockComponent = () => {

  return (
    <React.Fragment>
      <div></div>
    </React.Fragment>
  );
};

const MockComponentWrapped = withActiveFlag(MockComponent);

it(`withActiveFlag is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      isActive={false}
      onActiveChange={noop}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
