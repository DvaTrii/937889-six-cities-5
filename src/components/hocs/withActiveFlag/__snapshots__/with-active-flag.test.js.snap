// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveFlag is rendered correctly 1`] = `<div />`;

exports[`withAudio is rendered correctly 1`] = `null`;

exports[`withLoadFlag is rendered correctly 1`] = `null`;
